unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/DataStore.cs

```diff
@@ -67,5 +67,6 @@ public T Get<T>() where T : class, new()
         public DataStore_Rpc rpc => i.Get<DataStore_Rpc>();
         public DataStore_Channels channels => i.Get<DataStore_Channels>();
         public DataStore_WorldBlockers worldBlockers => i.Get<DataStore_WorldBlockers>();
+        public DataStore_LoadingScreen loadingScreen => i.Get<DataStore_LoadingScreen>();
```

Comment:
We should restrain from using these static shortcuts in new features. We could use `i.Get<>` directly, however I see how tedious it can get. 
Some time ago I introduced `Service<T>`, can you take a look maybe we can introduce something similar for `DataStore`s?

Reply:
Done!